### PR TITLE
adding public domain to readme.md

### PR DESCRIPTION
**Pull request by EricSchles**
Currently the readme for this repo doesn't have the public domain section in the readme.  adding it with this pull request

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,3 +46,12 @@ May also function as reference Service Provider implementation.
 
     openssl req -newkey rsa:2048 -nodes -keyout config/server.key \
       -x509 -out config/server.crt -config config/openssl.conf
+
+
+## Public domain
+
+This project is in the worldwide [public domain](LICENSE.md). As stated in [CONTRIBUTING](CONTRIBUTING.md):
```
Comment (sbc100) [inline README.md]: I don't really understand how this information is supposed to be split between README/CONTRIBUTING and LICENSE.

For sure LICENSE is the authoritative source, right?   So its good that README references LICENSE.   But why does it say "As stated in CONTIBUTING"?  Why not say "As stated in LICENSE"?  Or "as stated right here below" :)

Comment (sbc100) [inline README.md]: Also I think the title of this section should be "License", no?

**Comment by pkarman** [comment]
The main 18F readme template has the barest of license info, from which this PR was based: https://raw.githubusercontent.com/18F/open-source-policy/master/README_TEMPLATE.md

**Comment by sbc100** [comment]
Well fair enough I guress... but this project happens not have CONTRIBUTING file at all and it has LICENSE.txt not LICENSE.md.. so these links are both broken right now

**Comment by pkarman** [comment]
thanks @sbc100  - opened #34

**Pull request merged**